internal/runners/publish/publish.go

```diff
@@ -63,12 +65,19 @@ func New(prime primeable) *Runner {
 		graphql.WithHTTPClient(http.DefaultClient),
 		graphql.UseMultipartForm(),
 	)
-	client.EnableDebugLog()
 	client.SetTokenProvider(prime.Auth())
 	client.EnableDebugLog()
 	return &Runner{auth: prime.Auth(), out: prime.Output(), prompt: prime.Prompt(), project: prime.Project(), client: client}
 }
 
+type ParentIngredient struct {
+	IngredientID        strfmt.UUID
+	IngredientVersionID strfmt.UUID
+	Version             string
+	// dependencies
```

Comment:
Not sure how helpful this comment is... unless this was just meant to create some space in the struct definition.